src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private String name = "Circle";
```

Comment:
The 'name' field should not be a class variable since it is constant for all instances of Circle. Consider making it a final class-level constant, and follow the naming conventions for constants.